doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
```

Comment:
I think cloud-based may be a more accurate term.
https://www.techtarget.com/searchcloudcomputing/definition/cloud-application#:~:text=A%20key%20difference%20between%20cloud,a%20local%20computer%20or%20workstation.